server/pulp/server/managers/content/query.py

```diff
@@ -22,11 +22,6 @@ def list_content_types(self):
         """
         return content_types_db.all_type_ids()
 
-    def get_content_type(self, type_id):
```

Comment:
this isn't used anywhere, isn't documented, and obviously has no value based on the code you see below.